includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -304,12 +304,17 @@ public function delete( $order = null ) {
 		do_action( 'wpo_wcpdf_delete_document', $this, $order );
 	}
 
-	public function regenerate( $order = null ) {
+	public function regenerate( $data = null, $order = null ) {
```

Comment:
this is assuming nobody is currently using this function yet... it may be safer to keep the order of arguments in line with previous versions unless you have a good reason for needing `$data` first?